Sources/Recycling/ListViews/FavoriteFolders/Subviews/FavoriteFolderXmasView.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+public protocol FavoriteFolderXmasViewDelegate: AnyObject {
+    func favoriteFolderXmasViewDidTap(_ view: FavoriteFolderXmasView)
+}
+
+public final class FavoriteFolderXmasView: UIView {
+    public weak var delegate: FavoriteFolderXmasViewDelegate?
+
+    public var calloutText = "" {
+        didSet {
+            calloutView.isHidden = calloutText.isEmpty
+            calloutView.show(withText: calloutText)
+        }
+    }
+
+    private lazy var button: FloatingButton = {
+        let button = FloatingButton.favoritesXmasButton()
+        button.isUserInteractionEnabled = false
+        return button
+    }()
+
+    private lazy var calloutView: CalloutView = {
+        let view = CalloutView(direction: .down, arrowAlignment: .right(.veryLargeSpacing / 2 - .mediumSpacing))
```

Comment:
I think it's more readable if we just write `24` here instead, seeing as we have literals in the constraints below. What do you think?

Reply:
`.veryLargeSpacing` is the width of the button here, but I agree that `24` is better in this case 👍